src/main/java/pro/beam/api/http/BeamHttpClient.java

```diff
@@ -1,13 +1,14 @@
 package pro.beam.api.http;
 
+import com.auth0.jwt.internal.org.bouncycastle.util.encoders.Base64;
```

Comment:
Why import this, doesn't Java already have a Base64 parser? 
https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html